CHANGES.rst

```diff
@@ -25,13 +25,20 @@ cube_build
 ----------
 
 - Remove deleting the ``spaxel_dq`` array twice when using a weighting method of either msm or emsm. [#7586]
+  
+- Updated to read wavelength range for NIRSpec IFU cubes from the cubepars reference file,
+  instead of setting it based on the data. This makes use of new NIRSpec IFU cubepars reference
+  files with wavelength arrays for the drizzle method. [#7559]
```

Comment:
Why is this log message here?

Reply:
It had to be put in to remove conflicts, since this PR has been sitting around for a while as others have been merged, including at least one merger that had that log entry in the wrong place (so I moved it to the right place, under `cube_build`).